### PR TITLE
Update webhooks.md

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/webhooks.md [hunk 1]
```diff
@@ -1,59 +1,75 @@
...
+* **URL Method:** This is the HTTP method of the request that would be sent to the configured endpoint. Supported methods are `POST` and `GET`. By default, the `POST` method is used.
+
+* **Headers:** Add custom headers for your events via this option. These headers will be added to the request made from RudderStack to your webhook. By default, RudderStack adds the following headers for a `POST` request:
 
 |   Key | Value |
 | :--- | :--- |
 | `User-Agent` | `RudderLabs` |
 | `Content-Type` | `application/json` |
 
+{% hint style="info" %}
```
Comment (ameypv-rudder) [inline destinations/webhooks.md:59]: @arnab-p this is the newly added content - from L59 to L67.

File: destinations/webhooks.md [hunk 2]
```diff
@@ -1,59 +1,75 @@
...
+RudderStack also supports adding a dynamic header for your events through user transformation. For more information, refer to our 
+[**open-source transformer code**](https://github.com/rudderlabs/rudder-transformer/blob/5859c11d15abf779fd70a2fa45de1d754ef50e60/v0/destinations/webhook/transform.js#L53) in our GitHub repo.
```
Comment (arnab-p) [inline destinations/webhooks.md:61]: Can we move this transformation sample-user-transformation repository and add the links to that?

**Review by arnab-p — APPROVED** [review]

File: destinations/webhooks.md (hunk 2 above)
Comment (ameypv-rudder) [reply destinations/webhooks.md:61]: Sure, I'll do that.

Comment (ameypv-rudder) [reply destinations/webhooks.md]: This is done, and links added to the doc.

**Review by arnab-p — APPROVED** [review]

**Pull request merged**